lib/absinthe/schema.ex

```diff
@@ -107,13 +107,7 @@ defmodule Absinthe.Schema do
       @after_compile unquote(__MODULE__)
       @before_compile unquote(__MODULE__)
 
-      defdelegate __absinthe_type__(name), to: __MODULE__.Compiled
-      defdelegate __absinthe_directive__(name), to: __MODULE__.Compiled
-      defdelegate __absinthe_types__(), to: __MODULE__.Compiled
-      defdelegate __absinthe_types__(group), to: __MODULE__.Compiled
-      defdelegate __absinthe_directives__(), to: __MODULE__.Compiled
-      defdelegate __absinthe_interface_implementors__(), to: __MODULE__.Compiled
-      defdelegate __absinthe_prototype_schema__(), to: __MODULE__.Compiled
+      @schema_provider Absinthe.Schema.PersistentTerm
```

Comment:
This is only set this way temporarily, it should default back to the `Absinthe.Schema.Compiled` backend probably.